lib/katello/engine.rb

```diff
@@ -4,11 +4,13 @@ class Engine < ::Rails::Engine
 
     initializer 'katello.selective_params_parser', :before => :build_middleware_stack do |app|
       require 'katello/params_parser_wrapper'
-      app.middleware.swap(
-        ActionDispatch::ParamsParser,
-        Katello::ParamsParserWrapper,
-        ->(env) { env['PATH_INFO'] =~ /consumers/ && env['PATH_INFO'] =~ /profile|packages/ }
-      )
+      if Rails::VERSION::MAJOR == 4
+        app.middleware.swap(
+          ActionDispatch::ParamsParser,
+          Katello::ParamsParserWrapper,
+          ->(env) { env['PATH_INFO'] =~ /consumers/ && env['PATH_INFO'] =~ /profile|packages/ }
+        )
+      end
```

Comment:
I believe (and @ShimShtein can correct me), but i believe its used to prevent parsing of the body as json to conserve memory.  I'm guessing rails 5 is parsing this using some other mechanism, so we'd need to figure that out and prevent that from happening.